packages/select/test/pre-opened-polymer.test.js

```diff
@@ -0,0 +1,15 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync } from '@vaadin/testing-helpers';
+import '../vaadin-select.js';
+
+// NOTE: this test is Polymer only as it checks for Polymer specific bug.
+// Do not import `not-animated-styles.js` in this test intentionally, as
+// otherwise the immediately closed overlay would make it false positive.
+
+describe('pre-opened', () => {
+  it('should not throw error when adding a pre-opened select', () => {
+    expect(() => {
+      fixtureSync('<vaadin-select opened></vaadin-select>');
```

Comment:
I guess the issue is not about using and attribute specifically since this test would also fail with:
```js
const wrapper = fixtureSync('<div></div>');
const select = document.createElement('vaadin-select');
select.opened = true;
wrapper.appendChild(select);
```

Reply:
Right, the property also works. Renamed the PR.